x-pack/plugins/cases/public/common/lib/kibana/hooks.ts

```diff
@@ -104,29 +104,3 @@ export const useCurrentUser = (): AuthenticatedElasticUser | null => {
   }, [fetchUser]);
   return user;
 };
-
```

Comment:
This code is unused and shouldn't live in the cases plugin since each individual plugin that leverages cases will need to manage it's permissions separately.